.github/CODEOWNERS

```diff
@@ -339,6 +339,7 @@
 # Security Solution sub teams
 /x-pack/plugins/case @elastic/security-threat-hunting
 /x-pack/test/case_api_integration @elastic/security-threat-hunting
+/x-pack/test/case_api_no_sub_cases_integration @elastic/security-threat-hunting
```

Comment:
Once the sub cases feature is released we can delete these tests

Reply:
Is there any issue where we track what we should remove/enable when the sub cases feature is released?

Reply:
Good idea, I added it as an item here: https://github.com/elastic/kibana/issues/94115